### PR TITLE
gleam: 0.14.3 -> 0.14.4

### PR DESCRIPTION
**Pull request by happysalada**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -10408,6 +10408,7 @@ in
 
   gleam = callPackage ../development/compilers/gleam {
     inherit (darwin.apple_sdk.frameworks) Security;
+    inherit (darwin) libiconv;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:10411]:
```suggestion
```

not required.

Comment (happysalada) [reply pkgs/top-level/all-packages.nix:10411]: Do you mean it is not required in this file?

I had a failure with libiconv missing on the bot update PR, and so I just added it here. I verified locally that the failure was due to that lib missing. Do you mean to say that it works without this on your machine?

Comment (happysalada) [reply pkgs/top-level/all-packages.nix:10411]: @SuperSandro2000 Sorry to bother you with this, I need more details here.

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:10411]: > Do you mean it is not required in this file?

It shouldn't be required.

Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:10411]: Yeah, it is not required.

Comment (happysalada) [reply pkgs/top-level/all-packages.nix]: Ok I'll know for the future, thanks!

**Pull request merged**